rslib/src/text.rs

```diff
@@ -104,7 +104,19 @@ lazy_static! {
     pub static ref HTML_MEDIA_TAGS: Regex = Regex::new(
         r#"(?xsi)
             # the start of the image, audio, or object tag
-            <\b(?:img|audio|video|object)\b[^>]+\b(?:src|data)\b=
+            <\b(?:img|audio|video|object)\b
+
+            # any non-`>`, except inside `"` or `'`
+            (?:
+                [^>]
+            |
+                "(?:[^"]+?)"
+            |
+                '(?:[^']+?)'
```

Comment:
Do we need the inner (?: ... ) in these two clauses?

Reply:
You're right, we don't need them. Removed.